src/plugins/layout.ts

```diff
@@ -137,19 +189,50 @@ export default class Layout implements IPlugin {
           }
           break;
         case Source.Input:
+          layoutState = layoutState as IElementLayoutState;
```

Comment:
>layoutState = layoutState as IElementLayoutState; [](start = 10, length = 49)

Same

Reply:
Done

---
In reply to: [243076527](https://github.com/Microsoft/clarity-js/pull/116#discussion_r243076527) [](ancestors = 243076527)